rclcpp_lifecycle/include/rclcpp_lifecycle/state.hpp

```diff
@@ -87,7 +87,7 @@ class State
 protected:
   RCLCPP_LIFECYCLE_PUBLIC
   void
-  reset();
+  reset() noexcept;
```

Comment:
It's probably best to avoid backporting this change.

Reply:
Done